src/main/java/dev/pgm/events/ready/ReadyCommands.java

```diff
@@ -78,6 +79,25 @@ public void unreadyCommand(CommandSender sender, Match match, MatchPlayer player
     }
   }
 
+  @Command(aliases = "status", desc = "Display if teams are ready")
+  public void status(CommandSender sender, Match match) {
+    Stream<? extends Party> parties = match.getCompetitors().stream();
+    if (AppData.observersMustReady())
+      parties = Stream.concat(Stream.of(match.getDefaultParty()), parties);
+
+    parties
+        .map(
+            p ->
+                p.getColor()
+                    + p.getNameLegacy()
+                    + ChatColor.RESET
+                    + " is "
+                    + (readyParties.isReady(p)
+                        ? ChatColor.GREEN + "ready"
+                        : ChatColor.RED + "not ready"))
+        .forEach(sender::sendMessage);
```

Comment:
Events should have access to using `MatchPlayer` instead of `CommandSender`.

This means you can use the adventure library(#8) which will look something like this:
```suggestion
    Component readyComponent = text("ready", NamedTextColor.GREEN);
    Component notReadyComponent = text("not ready", NamedTextColor.RED);
    parties
        .map(
            p ->
                p.getName()
                .append(text(" is "))
                .append(readyParties.isReady(p) ? readyComponent : notReadyComponent))
          .forEach(player::sendMessage);
```

You should probably declare the ready/not ready component outside of this method.

Reply:
to be fair most of the plugin doesn't use adventure so maybe should hold that off until a future overhaul